source/documentation/basic-project.md

```diff
@@ -81,6 +81,62 @@ generating the site.
 With this option passed, `{{ site.url }}/about` will now be generated as
 `http://my.dev.host/blog-skelton/output_dev/about` instead of `/about`.
 
+## CSS and JavaScript
+
+The [Blog Skeleton](https://github.com/sculpin/sculpin-blog-skeleton) project
+uses [Symfony Webpack Encore](https://symfony.com/doc/current/frontend.html) to
+manage assets. If you're setting up your project with Sculpin directly, not
+using the skeleton, install Encore using the
+[setup instructions](https://symfony.com/doc/current/frontend/encore/installation.html).
+
+The rest of this section assumes that you are using Encore.
+
+Put your CSS and JS assets in the directory `source/assets/` as shown.
+
+    |-- source/
+    |  |-- assets/
```

Comment:
the top of the page is using a different layout for directories with backticks instead of vertical bars. i suggest we stick with the backticks

Reply:
I changed it to match the layout above. Looks to me like it's using vertical bars for children of a directory and then a backtick for the last child.

Reply:
There's a CLI tool called `tree` for generating this output. Might help. (It has "exclude" patterns that can help zero in on just the files and folders you want to show).